test_runlike.py

```diff
@@ -60,10 +60,10 @@ def test_udp_with_host_port_and_ip(self):
 
     def test_host_volumes(self):
         cur_dir = os.path.dirname(os.path.realpath(__file__))
-        self.expect_substr("--volume=%s:/workdir" % pipes.quote(cur_dir))
+        self.expect_substr("--volume %s:/workdir:rw" % pipes.quote(cur_dir))
 
     def test_no_host_volume(self):
-        self.expect_substr('--volume=/random_volume')
+        self.expect_substr(':/random_volume:rw')
```

Comment:
This test is for the `-v /random_volume` mount case. In this case, Docker will automatically generate a random volume to pair with `/random_volume`. Since the volume is random and cannot be fixed, you only need to check whether the second half exists.